transforms/helpers/parse-helper.js

```diff
@@ -129,10 +129,17 @@ function getDecoratorInfo(specifier, importPropDecoratorMap) {
   }
 
   const isMethodDecorator = METHOD_DECORATORS.includes(importedName);
+  const isComputedDecorator =
+    (importedName === "computed" && importPath === "@ember/object") ||
+    importPath === "@ember/object/computed" ||
+    importPath === "@ember/service" ||
+    importPath === "@ember/controller";
+
```

Comment:
Do we need to check against `@ember/object/evented` here?

Reply:
Hmm, can we get this value from `this.hasDecorators && !this.hasMethodDecorator`?

General question - Are `"@ember/service"` and `"@ember/controller"`considered as computed properties internally?

Reply:
I don't think we can since `@attribute`, `@className`, etc. are not method decorators but are also not computed properties, but I'm not as familiar with the code so maybe?

And yes, `@ember/service#inject` is just a CP under the hood